index.js

```diff
@@ -71,15 +69,14 @@ module.exports = fp(
 
         try {
             validateOptions({
-                environment,
-                release,
                 allowedStatusCodes,
                 onErrorFactory,
             });
         } catch (error) {
             return next(error);
         }
 
+        const { environment } = sentryOptions;
         const onError = onErrorFactory({
             environment,
```

Comment:
We should remove `environment` from here too

Reply:
Yes, seems like `defaultErrorFactory` is not interested in it at all. removed.